torchrl/envs/libs/pettingzoo.py

```diff
@@ -265,6 +270,13 @@ def _build_env(
     ):
         import pettingzoo
 
+        if importlib.metadata.version("pettingzoo") > "1.24.3":
```

Comment:
I would check if the version differs from 1.24.3 (above or below) and adapt the message accordingly